### PR TITLE
config-seed startup log WARN if expected directory missing

### PR DESCRIPTION
**Pull request by tsconn23**
#715 

Sample output
`WARN: 2018/11/07 15:42:44 open /Users/tconn/go/src/github.com/edgexfoundry/edgex-go/cmd/support-scheduler/res/configuration.toml: no such file or directory`

Signed-off-by: Trevor Conn <trevor_conn@dell.com>

### CODE REVIEW

File: internal/seed/config/populate.go [hunk 1]
```diff
@@ -97,28 +97,32 @@ func ImportConfiguration(root string, profile string) error {
 
 		LoggingClient.Debug("reading toml " + path)
 		// load the ToML file
-		config, _ := toml.LoadFile(path)
+		config, err := toml.LoadFile(path)
+		if err != nil {
+			LoggingClient.Warn(err.Error())
```
Comment (anonymouse64) [inline internal/seed/config/populate.go]: Is there a reason we don't also propagate this error here with `return err`?

Comment (anonymouse64) [reply internal/seed/config/populate.go]: Oh also, if you do return the error here, you should drop the else block and re-outdent the else block (in accordance with golint)

Comment (tsconn23) [reply internal/seed/config/populate.go]: If I propagate the error then the loop stops. You said in your comment you wanted the rest of the configs to be populated.

Comment (anonymouse64) [reply internal/seed/config/populate.go]: Ah yes you're right sorry I thought ImportConfiguration was run in a loop somewhere, but I see that it runs the loop itself.

**Pull request merged**